Readme.md

```diff
@@ -229,6 +229,19 @@ class FooHydrator2 implements HydratorInterface
 }
 ```
 
+## SerializerRouter
+
+If you dispatch/consume messages serialized with `Happyr\MessageSerializer\Serializer` and default Symfony messenger to same 
+transport you might wanna use `Happyr\MessageSerializer\SerizlizerRouter`. This serializer will decide weather it will 
+use `Happyr\MessageSerializer\Serializer` to decode/encode your message or the default one from Symfony messenger. 
```

Comment:
Lines in docs should not be too long. Make a line break after the first word that exceeds 80 chars.